src/Model/OutputResponse.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Valantic\PimcoreFormsBundle\Model;
+
+class OutputResponse
+{
+    /**
+     * @var array<AbstractMessage>
+     */
+    private array $messages;
```

Comment:
Should this be initialized with an empty array to prevent access to an uninitialized property?